core/api/transport/impl/ws/ws_listener_impl.cpp

```diff
@@ -8,6 +8,11 @@
 #include <boost/asio.hpp>
 #include "application/app_state_manager.hpp"
 
+namespace {
+  constexpr const char *kOpenedRpcSession = "kagome_rpc_sessions_opened";
```

Comment:
I think it might be more informative to suffix these constants with something like _MetricsFamily, because otherwise their names can be confusing,